dev-python/mamba/mamba-0.8.6-r1.ebuild

```diff
@@ -9,7 +9,7 @@ inherit distutils-r1
 
 DESCRIPTION="... testing tool ... Born under the banner of Behavior Driven Development"
 HOMEPAGE="http://nestorsalceda.github.io/mamba"
-SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+SRC_URI="https://github.com/nestorsalceda/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please rename the distfile to `${P}.gh.tar.gz` or something alike, so the mirrors will be updated.

Reply:
Fixed. Thanks. Was unsure if something like this was necessary.